cluster-autoscaler/Makefile

```diff
@@ -35,7 +35,7 @@ build-binary: clean deps
 	$(ENVVAR) GOOS=$(GOOS) go build -o cluster-autoscaler ${LDFLAGS_FLAG} ${TAGS_FLAG}
 
 test-unit: clean deps build
-	GO111MODULE=off go test --test.short -race ./... ${TAGS_FLAG}
+	go test --test.short -race ./... ${TAGS_FLAG}
```

Comment:
Seems `-mod=vendor` will be enabled by default. Copy from `go help modules` commands:
```
If the vendor directory is present in the main module's root directory, it will
be used automatically if the "go" version in the main module's go.mod file is
1.14 or higher. Build commands like 'go build' and 'go test' will load packages
from the vendor directory instead of accessing the network or the local module
cache. To explicitly enable vendoring, invoke the go command with the flag
-mod=vendor. To disable vendoring, use the flag -mod=mod.
```

Reply:
Good to know, thanks!